### PR TITLE
Documentation improvements. Remove deprecated annotation. Add note for Translator changes.

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: stylesheets/stylesheet.css [hunk 1]
```diff
@@ -219,7 +219,7 @@ Full-Width Styles
 
 .inner {
   position: relative;
-  max-width: 640px;
+  max-width: 75%;
```
Comment (rnc) [inline stylesheets/stylesheet.css:222]: @dwalluck I think we should apply this to GME as well. Can you do this please?

Comment (dwalluck) [reply stylesheets/stylesheet.css:222]: Sure. I suppose that fixed width makes more sense on smaller devices. You can see it used elsewhere, particularly for other media types. For a large screen, I am not sure why there is such a strict limit.

Comment (dwalluck) [reply stylesheets/stylesheet.css:222]: https://github.com/project-ncl/gradle-manipulator/pull/296

Comment (rnc) [reply stylesheets/stylesheet.css:222]: Thanks!

**Comment by rnc** [comment]
> Pending the official API name change, of course. Do we the ticket numbers on the first line?

Sorry not sure what you mean re ticket numbers - I have referenced the JIRAs in the commits.

**Pull request merged**